src/renderer/store/certificates/certificates.saga.test.ts

```diff
@@ -0,0 +1,38 @@
+import { expectSaga } from 'redux-saga-test-plan'
+
+import { certificatesActions, certificatesReducer, CertificatesState } from './certificates.reducer'
+import { creactOwnCertificate } from './certificates.saga'
+import { createUserCsr } from '../../pkijs/generatePems/requestCertificate'
+import { createUserCert } from '../../pkijs/generatePems/generateUserCertificate'
+
+describe('checkCertificatesSaga', () => {
+  test('creating own cert', () => {
+    void expectSaga(creactOwnCertificate, { payload: 'name', type: certificatesActions.creactOwnCertificate.type })
+      .withReducer((certificatesReducer), {
```

Comment:
Shiyld be:

combineReducers({ [StoreKeys.certificates]: certificatesReducers })

This way you have the same structure in tests as in app, so your selectors work normally